audio_driver/audio_driver.py

```diff
@@ -214,23 +207,19 @@ def bus_call(self, bus, msg):
                         if self._current_audio.loop_delay != 0:
                             # Wait the next playback
                             #   for the time set in "Loop Delay".
-                            asyncio.set_event_loop(self._loop)
-
-                            # [Remaining issue] Asynchronization using "asyncio" 
-                            #   is assumed, but it is not asynchronous processing.
-                            #   The current implementation is equivalent to 
-                            #   the implementation using "sleep".
-                            self._loop.run_until_complete(
-                                self.wait_with_delay(self._current_audio.loop_delay))
+                            time.sleep(self._current_audio.loop_delay)
+
                         self.gst_play()
 
             self._lock.release()
         elif msg.type == Gst.MessageType.ERROR:
+            self._lock.acquire()
```

Comment:
Due to the complexity of the implementation, it is implemented to lock to the outermost side.